spec/support/features/book_helper.rb

```diff
@@ -0,0 +1,8 @@
+module Features
+  def create_book(book_title, book_author)
```

Comment:
`book_title` and `book_author` are probably more specific than they need to be. You can probably do `title` and `author` and it would make perfect sense still. What do you think?